start-mongodb.sh

```diff
@@ -18,6 +18,17 @@ if [ -z "$MONGODB_VERSION" ]; then
 fi
 
 
+echo "::group::Selecting correct MongoDB client"
+if [ "`echo $MONGODB_VERSION | cut -c1-1`" = "4" ]; then
```

Comment:
@ecarruda Do you know if we can just use `1` instead of `1-1` for the first character?

```suggestion
if [ "`echo $MONGODB_VERSION | cut -c 1`" = "4" ]; then
```

Otherwise, I think a space between `-c` and `1-1` would be nice to increase readability. Well, hopefully my requested space doesn’t break anything.

Reply:
Let me do some tests here

Reply:
You are right @marcuspoehls thanks for this tip, checked on cut man page about this syntax and using -c alone (without --complement) we can improve this syntax as you suggested, changed it :)